src/data/screenshots.json

```diff
@@ -219,201 +219,201 @@
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_valid_details_part1", "alt": "Recovery certificate image 2 of 3" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_valid_details_part2", "alt": "Recovery certificate image 3 of 3" }
                 ]
             },
             {
                 "title": "Check Certificates for Validity Before a Trip",
                 "anchor": "ios_check_validity_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Certificate check image 1 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_validation_date_selection", "alt": "Certificate check image 2 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_validation_country_selection", "alt": "Certificate check image 3 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_validation", "alt": "Certificate check image 4 of 4" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Certificate check image 1 of 4" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_certificate_validation_date_selection", "alt": "Certificate check image 2 of 4" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_certificate_validation_country_selection", "alt": "Certificate check image 3 of 4" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_certificate_validation", "alt": "Certificate check image 4 of 4" }
                 ]
             },
             {
                 "title": "Invalid Certificates",
                 "anchor": "ios_invalid_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Invalid certificate image 1 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part1", "alt": "Invalid certificate image 2 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part2", "alt": "Invalid certificate image 3 of 3" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Invalid certificate image 1 of 3" },
```

Comment:
This screenshot has not been taken on version 2.11. It should be removed, as the screenshot which follows is just a newer version of it. 

```suggestion
```

**Don't forget to change the counting "Invalid certificate image x of 2"**

Reply:
@Ein-Tim Thanks!